source/_integrations/utility_meter.markdown

```diff
@@ -213,12 +213,12 @@ automation:
       variables:
         tariff: "offpeak"
   action:
-    - service: select.select_option
+    - service: input_select.select_option
```

Comment:
This change is not correct. This refers to the select entity that is provided by the utility meter.

../Frenck

Reply:
huh?
I just checked again - and it seems, that this depends on the selected language in the user-profile....

english:
![image](https://github.com/home-assistant/home-assistant.io/assets/26391061/5d9c84f1-ceb2-4d5d-85df-d18858d19d56)

german:
![image](https://github.com/home-assistant/home-assistant.io/assets/26391061/7e2e55f6-1cdc-48eb-bb4b-efde7185dadb)

Reply:
ok, now I don't understand why I am getting both options... in english and german.
![image](https://github.com/home-assistant/home-assistant.io/assets/26391061/59f7a0d7-1808-4003-bdde-3f60dc718742)

When I set up the automation, I got the error, that the service 'select.select_option` could not be found... this is why I searched the services and found the `input_select` ...

Now, I tried to run the automation once with english and once with german language - and both times, I didn't got any error (maybe, because other conditions are not met...)
But now, I do have both `select.select_option` and `input_select.select_option` in the service list.

Reply:
Hey @frenck, sorry for tagging you directly.
Can you please have a look, if this issue with the service namings might be an issue when switching the language?

I'll try to provide some more information about this and verify the behave